src/server/cluster/cluster_slot_migration.h

```diff
@@ -9,11 +9,20 @@ namespace dfly {
 
 class ClusterSlotMigration : ProtocolClient {
  public:
+  enum State : uint8_t { C_NO_STATE, C_CONNECTING, C_FULL_SYNC, C_STABLE_SYNC } state_ = C_NO_STATE;
```

Comment:
state_ should be private

Reply:
I would like to leave it public, because I want to provide status with this state